src/gpgservice.py

```diff
@@ -0,0 +1,69 @@
+import gnupg
+import json
+import pprint
+import os
+
+class GPGService:
+
+	def __init__(self):
+		super(GPGService, self).__init__()
+		self.gpg = gnupg.GPG(gnupghome ='homegpg')
+		with open('keys.json','a+') as dataFile:
+			if os.stat('keys.json').st_size is not 0:
```

Comment:
EAFP: https://docs.python.org/3/glossary.html

Here you're:

1. creating a file if it doesn't exists
2. checking the file size (using path instead of already opened file descriptor, which is a problem on itself as it can cause a race condition)
3. loading file if it's not empty

In Python you're usually asking for forgiveness, not for permission, which means that you should just open the file read-only and read it. If it doesn't exist it will throw FileNotFoundError exception, which you should just catch. Same thing for invalid JSON in file.